demisto_sdk/commands/format/update_generic.py

```diff
@@ -220,95 +219,58 @@ def regex_matching_key(self, field, schema_keys):
                 return reg
         return None
 
-    def set_fromVersion_of_generic_object(self, from_version=None):
-        """Sets fromVersion key in a generic object file:
-        Args:
-            from_version: The specific from_version value.
-        """
-        if self.verbose:
-            click.echo('Setting fromVersion field of a generic object')
-        # If user entered specific from version key to be set
-        if from_version:
-            if LooseVersion(from_version) < LooseVersion(GENERIC_OBJECTS_DEFAULT_FROMVERSION):
-                click.echo(f'The given fromVersion value for generic entities should be'
-                           f' {GENERIC_OBJECTS_DEFAULT_FROMVERSION} or above , given: {from_version}.\n'
-                           f'Setting fromVersion field to {GENERIC_OBJECTS_DEFAULT_FROMVERSION}')
-                self.data[self.from_version_key] = GENERIC_OBJECTS_DEFAULT_FROMVERSION
-            else:
-                self.data[self.from_version_key] = from_version
+    def get_answer(self, promote):
+        click.secho(promote,
+                    fg='red')
+        return input()
+
+    def ask_user(self):
+        user_answer = self.get_answer(
+            'Either no fromversion is specified in your file,'
+            ' or it is lower than the minimal fromversion for this content type, would you like to set it to the default? [Y/n]')
+        if user_answer in ['Y', 'y', 'yes', 'Yes']:
+            return True
         else:
-            if LooseVersion(self.data.get(self.from_version_key, DEFAULT_CONTENT_ITEM_FROM_VERSION)) < \
-                    LooseVersion(GENERIC_OBJECTS_DEFAULT_FROMVERSION):
-                self.data[self.from_version_key] = GENERIC_OBJECTS_DEFAULT_FROMVERSION
+            click.secho('Skipping update of fromVersion', fg='yellow')
+            return False
+
+    def set_default_from_version(self, default_from_version: str, current_fromversion_value: Optional[str] = None, file_type: Optional[str] = None):
```

Comment:
please add a docstring explaining the logic (instead of the comments). but great job here